app/src/components/v-image.vue

```diff
@@ -25,13 +25,16 @@ const srcData = ref<string>(emptyPixel);
 
 let loaded = false;
 
-const observer = new IntersectionObserver(async (entries) => {
+const observer = new IntersectionObserver((entries) => {
 	if (entries.length === 0) return;
-	inView.value = entries[0].isIntersecting;
 
-	if (entries[0].isIntersecting && !loaded && props.src) {
-		loadImage();
-	}
+	entries.forEach((entry) => {
+		inView.value = entry.isIntersecting;
```

Comment:
Is there ever a case where the second entry doesn't equal the first entry? If so, we might have to make this a little smarter, as the second entry can override the state of the first entry

Reply:
Yeah that's a great point. They are definitely not equal, with one of them being the chrome-specific one where it has 0 intersecting value and no root bounds:

![chrome_4hZZGAhFJA](https://user-images.githubusercontent.com/42867097/185109907-23bb5dcf-ddad-4a2b-9b34-96c999aa3e4a.png)

Thoughts on using the following logic with `.some()` instead? That way there shouldn't be overriding as long as one of them is true. Tested it and it does work:

```js
	const isIntesecting = entries.some((entry) => entry.isIntersecting);
	inView.value = isIntesecting;

	if (isIntesecting && !loaded && props.src) {
		loadImage();
	}
```

Reply:
@azrikahar That first one is confusing. It's the same `img.image`, but with 0x0 dimensions? Can you confirm that's the same element reference?

Reply:
Yeah somehow it is the same element, down to the exact same base64 `src`, `alt` etc. They even have the same `x`, `y`, `height`, `width` 🤔 but oddly the IntersectionObserverEntry properties can have this disparity.

Reply:
@azrikahar Is the "correct" one always the second/last one?

Reply:
Ya seems like it is from what I observed 👍

![image](https://user-images.githubusercontent.com/42867097/185169735-c63ed559-2eff-4e75-b169-67d334424658.png)

Reply:
OK! In that case, should we start with an `entries.at(-1)` instead of looping over them?

Reply:
Sounds good! Tested and works well. Pushed the change 👍